src/main/java/ua.horobets/hw20/Builder/Computer.java

```diff
@@ -0,0 +1,28 @@
+package ua.horobets.hw20.Builder;
+
+import lombok.*;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
You need to implement this pattern without using lombok @Builder annotation)
But it's good that you know about it :)